build.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.7
+#!/usr/bin/env python
```

Comment:
I probably made this change out of habit. I can change it back.

Reply:
I ask because there's an explicit check for python 2.x at the beginning of the script: https://github.com/LLK/scratch-blocks/pull/1471/files#diff-2955d5257f635de1df53f55a171ca5c7R39